./gradlew build fails because of gradle-node-plugin not detecting local npm/node

Executing `./gradlew build` results in the following error:

```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':scenarioo-client:npmInstall'.
> A problem occurred starting process 'command 'npm''

* Try:
Run with --info or --debug option to get more log output.

* Exception is:
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':scenarioo-client:npmInstall'.
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:69)
...
   org.gradle.launcher.daemon.server.DaemonStateCoordinator$1.run(DaemonStateCoordinator.java:293)
        at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:54)
        at org.gradle.internal.concurrent.StoppableExecutorImpl$1.run(StoppableExecutorImpl.java:40)
Caused by: org.gradle.process.internal.ExecException: A problem occurred starting process 'command 'npm''
        at org.gradle.process.internal.DefaultExecHandle.setEndStateInfo(DefaultExecHandle.java:198)
        at org.gradle.process.internal.DefaultExecHandle.failed(DefaultExecHandle.java:329)
        at org.gradle.process.internal.ExecHandleRunner.run(ExecHandleRunner.java:86)
        ... 2 more
Caused by: net.rubygrapefruit.platform.NativeException: Could not start 'npm'
        at net.rubygrapefruit.platform.internal.DefaultProcessLauncher.start(DefaultProcessLauncher.java:27)
        at net.rubygrapefruit.platform.internal.WrapperProcessLauncher.start(WrapperProcessLauncher.java:36)
        at org.gradle.process.internal.ExecHandleRunner.run(ExecHandleRunner.java:68)
        ... 2 more
Caused by: java.io.IOException: Cannot run program "npm" (in directory "/Users/chrgro/Documents/Code/scenarioo/scenarioo-client"): error=2, No such file or directory
        at net.rubygrapefruit.platform.internal.DefaultProcessLauncher.start(DefaultProcessLauncher.java:25)
        ... 4 more
Caused by: java.io.IOException: error=2, No such file or directory
        ... 5 more
```

There is an issue in the gradle-node-plugin repo https://github.com/srs/gradle-node-plugin/issues/254

Problem is that the local node/npm install is not correctly detected. We could solve this as suggested in the issue by downloading a specific node version.

--- a/scenarioo-client/build.gradle
+++ b/scenarioo-client/build.gradle
@@ -33,6 +33,11 @@ buildscript {
     }
 }
 
+node {
+    version = '8.9.4'
+    download = true
+}
+
 clean {
     delete "dist"
 }

